Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/colmartech_vendors.yml

```diff
@@ -1948,12 +1948,20 @@
   - type: CMAutomatedVendor
     job: CMMilitaryPolice
     sections:
+    - name: Police Set (Mandatory)
+      entries:
+      - id: RMCVendorBundleMilitaryPoliceApparel
     - name: Standard Equipment
       takeAll: CMStandard
       entries:
-      - id: RMCVendorBundleMilitaryPoliceApparel
+      - id: CMHandsBlackMarine
+      - id: CMHeadsetMPO
+      - id: CMBootsBlackFilled
+    - name: Armor
+      takeAll: CMStandard
```

Comment:
`takeAll: CMStandard`
Should be 
`takeAll: Armor`
Otherwise you can only take either the standard equipment or armor, these are IDs that prevent you from taking more from a different vendor